internal/manifest/v1alpha1/custom_resource.go

```diff
@@ -0,0 +1,75 @@
+package v1alpha1
+
+import (
+	"context"
+
+	manifestv1alpha1 "github.com/kyma-project/module-manager/api/v1alpha1"
+	declarative "github.com/kyma-project/module-manager/pkg/declarative/v2"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const CustomResourceManager = "resource.kyma-project.io/finalizer"
+
+func PostRunCreateCR(
```

Comment:
could we  add a comment to this function to quickly understand which resource is getting created?